pkg/console/subresource/configmap/brand_custom_logo.go

```diff
@@ -0,0 +1,23 @@
+package configmap
+
+// borrowed from the image package
+// image.RegisterFormat()
+var commonImageHeaders = []string{
+	"\xff\xd8\xff",      // "image/jpeg"
+	"\x89PNG\r\n\x1a\n", // "image/png"
+	"GIF87a",            // "image/gif"
+	"GIF89a",            // "image/gif"
+}
+
+// TODO: we could check for actual image headers in the binary to ensure its actually an image, if we really wanted to :)
+// tests if bytes provided are likely an image by reading the
+// first set of bytes and checking against some well known
+// image headers.
+func IsLikelyCommonImageFormat(bytes []byte) bool {
```

Comment:
I'd like to implement this func, but it is not strictly necessary.  We decided we would mount the image no matter what, so long as the configmap exists & the config is provided.

This would be helpful just to notify the user that perhaps the image is not correct.  We wouldn't error & die, but we could give a status.condition update.

Reply:
Meaning this should probably not hold up the PR, but is a nice to have.